src/isIsogram.test.js

```diff
@@ -6,4 +6,20 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  test('should return true if each letter occurs only once', () => {
+    expect(isIsogram('playgrounds')).toBe(true);
+  });
+
+  test('empty string should return true', () => {
+    expect(isIsogram('')).toBe(true);
+  });
+
+  test('should return false if there is duplication of letters', () => {
+    expect(isIsogram('look')).toBe(false);
+  });
+
+  test('function should be case-insensitive (M and m are the same)', () => {
```

Comment:
```suggestion
  test('should be case-insensitive', () => {
```
In your test data there is just one letter 'm', so the comment about 'M' and 'm' doesn't make sense